src/cozy/react/components/InstallationPage/index.jsx

```diff
@@ -42,7 +44,9 @@ const STEPS = {
   configureExtension: 3
 }
 
-function getStepContent(step, setActiveStep, { hasHint, onSkipExtension }) {
+const COZY_BAR_HEIGHT = 48
```

Comment:
nit: Don't we have a more "future proof" option here? I suppose we won't change the cozy-bar style in a near future, but wondering if we can ensure we won't be able to forget this line if we do it. (but this may be premature optimisation)

Reply:
We have this harcorded value in a lot of place in the cozy code base, for instance: 
https://github.com/cozy/cozy-ui/blob/8851fc1ed6ae9593ba6ffce77b7a077d7c8cabd7/react/UploadQueue/styles.styl#L9
https://github.com/cozy/cozy-notes/blob/e7af593b9fb94314af59d05efb105cd00c9450ce/src/components/notes/editor-view.styl#L8
https://github.com/cozy/cozy-notes/blob/e7af593b9fb94314af59d05efb105cd00c9450ce/src/components/notes/editor-view.styl#L8
https://github.com/cozy/cozy-banks/blob/7d9638271f1290ce861a4206d02005893c5a38ef/src/ducks/balance/NoAccount.styl#L8
https://github.com/cozy/cozy-drive/blob/190627d84cde133ad2f92ad7b3858e7385041201/src/photos/ducks/upload/styles.styl#L134


Maybe the solution is to have to cozy-bar writing by itself a --cozy-bar-height in order to be able to use it. 

So no, we don't have ATM an other way to do that